netlogo-core/src/main/api/LabDefaultThreads.scala

```diff
@@ -0,0 +1,10 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.api
+import scala.math.floor
+
+object LabDefaultThreads {
+ val RATIO = 0.75
+ // Determines the number of threads in BehaviorSpace if the user has not specified a value
+ def getLabDefaultThreads :Int  = { floor(Runtime.getRuntime.availableProcessors * RATIO).toInt }
```

Comment:
Nit: No space before `:` for type specifier, prefer space after `:` type specifier, no space before `=`.